### ISSUE TITLE
When voicelines are disabled, Monitor Screens should display generic Aperture logo as shown on the item's icon

### ISSUE THREAD

**Issue by ncrecc** [BEEmod/BEE2-items#3595]
Currently Monitor Screens are entirely black when voicelines are disabled. This would make it less awkward to use aesthetically.

**Comment by DeliWasTaken** [comment]
I agree, this might look pretty aesthetically pleasing in a some cases. But not when voicelines are disabled, also for when the announcer's, cave's, and caroline's lines are being used, or any other non-seen character that I'm forgetting.
